examples/phabricator/v1beta3/phabricator-controller.json

```diff
@@ -0,0 +1,52 @@
+{
+  "kind": "ReplicationController",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "phabricator-controller",
+    "labels": {
+      "name": "phabricator"
+    }
+  },
+  "spec": {
+    "replicas": 1,
+    "selector": {
+      "name": "phabricator"
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "name": "phabricator"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "phabricator",
+            "image": "fgrzadkowski/example-php-phabricator",
+            "ports": [
+              {
+                "name": "http-server",
+                "containerPort": 80
+              }
+            ],
+            "env": [
+              {
+                "name": "MYSQL_SERVICE_IP",
+                "value": "173.194.242.66"
+              },
+              {
+                "name": "MYSQL_SERVICE_PORT",
+                "value": "3306"
+              },
+              {
+                "name": "MYSQL_PASSWORD",
+                "value": "1234"
+              }
+            ],
+            "imagePullPolicy": "Always"
```

Comment:
This is the default value and hence can be deleted to keep the example short and simple.